src/editor/components/components/ManagedStreetSidebar.js

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import PropertyRow from './PropertyRow';
+
+const ManagedStreetSidebar = ({ entity }) => {
+  const componentName = 'managed-street';
+  // Check if entity and its components exist
+  const component = entity?.components?.[componentName];
+
+  return (
+    <div className="managed-street-sidebar">
+      <div className="street-controls">
+        <div className="details">
+          {component && component.schema && component.data && (
+            <>
+              <PropertyRow
+                key="length"
+                name="length"
+                label="Street Length"
+                schema={component.schema['length']}
```

Comment:
Just use the dot syntax component.schema.length. Same for the other ['key'] below.

Reply:
6ab92861bdcd2b4529a2b4d53a4c0e98aa856d12